events.css

```diff
@@ -0,0 +1,29 @@
+article section: first-child {
+	font-weight: bolder;
+}
+
+article section: last-child {
```

Comment:
Same thing here.  Be sure to delete the white space after the ":", as this was causing the styling to not show up on the last child of Section.